sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/models/StorageAccount.java

```diff
@@ -547,6 +561,45 @@ interface WithHns {
             WithCreate withHnsEnabled(boolean enabled);
         }
 
+        /** The stage of storage account definition allowing to configure allow cross tenant replication. */
+        interface WithAllowCrossTenantReplication {
+            /**
+             * Allows to allow cross tenant replication, configured by individual containers.
+             *
+             * @return the next stage of storage account definition
+             */
+            WithCreate enableAllowCrossTenantReplication();
+
+            /**
+             * Disables allow cross tenant replication
+             *
+             * Disabling in storage account overrides the allow cross tenant replication settings for individual containers.
+             *
+             * @return the next stage of storage account definition
+             */
+            WithCreate disableAllowCrossTenantReplication();
```

Comment:
Would you let us know whether the default(if `null` is specified) is true or false?

If true, then we probably only need `disableAllowCrossTenantReplication()` in create phase. Otherwise, we only need `enableAllowCrossTenantReplication()` in create phase. (update mode need both no matter what)

Reply:
I rechecked the status of the `Storage Account` in the `Azure Portal`. The latest version is now consistent with the initial settings and changes of status in `Azure Portal`.

Reply:
The default status of `AllowCrossTenantReplication` is true, so only keep `disableAllowCrossTenantReplication()` in create.
The default status of `defaultToOAuthAuthentication` is false, so only keep `enableDefaultToOAuthAuthentication()` in create.